sui/utils/index.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
I prefer to not flatten it so it's more clear where the import comes from, but not a strong opinion if you think this is more standard

Reply:
It can be commonly found in many js projects, but it's not a strong pattern. We need to compare the trade-offs for each project. I agree that it's a bit less clear to find the source file for imported method. However, In VSCode, if you click at the imported method, it will redirect you to the original file, not the index file. That also works for some methods in GitHub as well.

But the benefits that I think it's worth to add is, when you're working on a script, and you want to import some utils functions, you can get all available options from autocomplete in the `require('./utils')` statement. This way, you don't have to remember where the function is located.